test/rfxcom.spec.js

```diff
@@ -4,164 +4,175 @@ var rfxcom = require('../lib'),
     matchers = require('./matchers'),
     protocols = rfxcom.protocols;
 
-describe("RfxCom", function() {
-    beforeEach(function() {
+describe("RfxCom", function () {
+    beforeEach(function () {
         this.addMatchers({
             toHaveSent: matchers.toHaveSent
         });
     });
 
-    describe("RfxCom class", function() {
-        describe("data event handler", function() {
-            it("should emit a response message when it receives message type 0x02", function(done) {
+    describe("RfxCom class", function () {
+        describe("data event handler", function () {
+            it("should emit a response message when it receives message type 0x02", function (done) {
                 var fakeSerialPort = new FakeSerialPort(),
-                    device = new rfxcom.RfxCom("/dev/ttyUSB0", {
+                        device = new rfxcom.RfxCom("/", {
```

Comment:
That attempt wasn't too successful. I ended up having to add flags in various places so the calling code would know how to handle a 'disconnect' event when the RFXtrx433 wasn't in the connected state. I've pushed the original change for you to see the code in context.